docs/guide/sending-email.md

```diff
@@ -0,0 +1,283 @@
+
+### Example for Mailgun
+
+<AposCodeBlock>
+
+```js
+const mg = require('nodemailer-mailgun-transport');
+
+const auth = {
+  auth: {
+    // key supplied with your Mailgun account
+    api_key: 'key-1234123412341234',
+    //one of your domains listed at your https://app.mailgun.com/app/sending/domains
+    domain: 'sandbox.domain.com'
+  }
+};
+
+module.exports = {
+  options: {
+    nodemailer: mg(auth);
+  }
+};
+
+```
+<template v-slot:caption>
+/modules/@apostrophecms/email
+</template>
+
+</AposCodeBlock>
+
+## Sending email from a module
+Once the `@apostrophecms/email` module is configured, email can be sent from any module using the `self.email(req, template, data, options)` method.
+
+The first parameter passed to this method is the `req`.
+
+The next parameter, `template`, takes the name of a Nunjucks template that will make up the body of the email. This template should be located in the `views` template of the module. The method will pass this HTML template, as well as an automatically generated plain text version, to the `nodemailer` transport object.
+
+The `data` parameter takes an object that will be passed to the Nunjucks template for populating any customized fields. It can be accessed through `data.property` within the template.
+
+The final parameter, `options`, should be an object that contains the information for the email header. This is typically `from`, `to`, and `subject`. Any of these can have defaults set in the `options` of the `@apostrophecms/email` module, just like the transport. Any parameters specified in an individual method call will override those set in this manner.
+
+### Example usage
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    lable: 'article',
+    pluralLabel: 'articles'
+  },
+  // additional module code
+  handlers(self) {
+    return {
+      'afterSave': {
+        async sendEmail(req, piece) {
+          const options = {
+            from: 'admin@mysite.com',
+            to: 'editors@mysite.com',
+            subject: 'New Article added'
+          };
+          try {
+            await self.email(req, 'email.html', { piece }, options);
+          } catch (err) {
+            self.apos.util.error('email notification error: ', err);
+          }
+        }
+      }
+    }
+  }
+};
+```
+
+<template v-slot:caption>
+/modules/article/index.js
+</template>
+
+</AposCodeBlock>
+
+
+<AposCodeBlock>
+
+```django
+<h1>A new article has been added to the site</h1>
+<p>Here is the blurb</p>
+{% data.piece.title %}
```

Comment:
data.piece feels good to me, but you changed the email method call examples back to passing piece by itself, so I would expect data.piece not to exist at this point.

Recommend you go back to that nice `{ data: piece }` setup.

Reply:
Excuse me, `{ data: { piece } }`